### PR TITLE
i18n(zh-cn): Update some zh-cn pages about v5

### PR DESCRIPTION
**Pull request by liruifengv**
#### Description (required)

- https://github.com/withastro/docs/pull/10376
- https://github.com/withastro/docs/pull/10401
- https://github.com/withastro/docs/pull/10409
- https://github.com/withastro/docs/pull/10460
- https://github.com/withastro/docs/pull/10459
- https://github.com/withastro/docs/pull/10461
- https://github.com/withastro/docs/pull/10434
- https://github.com/withastro/docs/pull/10475
- https://github.com/withastro/docs/pull/10506
- https://github.com/withastro/docs/pull/10504
- https://github.com/withastro/docs/pull/10505
- https://github.com/withastro/docs/pull/10507
- https://github.com/withastro/docs/pull/10513
- https://github.com/withastro/docs/pull/10512
- https://github.com/withastro/docs/pull/10514
- https://github.com/withastro/docs/pull/10516
- https://github.com/withastro/docs/pull/10503
- https://github.com/withastro/docs/pull/10511
- https://github.com/withastro/docs/pull/10520
- https://github.com/withastro/docs/pull/10518
- https://github.com/withastro/docs/pull/10517
- https://github.com/withastro/docs/pull/10527
- https://github.com/withastro/docs/pull/10526
- https://github.com/withastro/docs/pull/10525
- https://github.com/withastro/docs/pull/10523
- https://github.com/withastro/docs/pull/10528
- https://github.com/withastro/docs/pull/10529

#### Related issues & labels (optional)

- https://github.com/withastro/docs/pull/9240
- https://github.com/withastro/docs/pull/10447

### CODE REVIEW

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-gatsby.mdx [hunk 1]
```diff
@@ -119,11 +121,11 @@ Gatsby 和 Astro 共享一些相似性，这将帮助你迁移你的项目：
 
 3. 将所有必要的 JavaScript，包括 import 语句，都放入["代码围栏" (`---`)](/zh-cn/basics/astro-components/#组件脚本)中。注意：在 Astro 中，经常会直接在 HTML 模板内编写[条件渲染内容](/zh-cn/reference/astro-syntax/#动态-html)的 JavaScript。
 
-4. 使用[`Astro.props`](/zh-cn/reference/api-reference/#astroprops)来访问之前传递给你的 Gatsby 函数的任何其他 props。
+4. 使用[`Astro.props`](/zh-cn/reference/api-reference/#props)来访问之前传递给你的 Gatsby 函数的任何其他 props。
 
 5. 决定是否需要将导入的组件也转换为 Astro。如果已经安装了官方的 React 集成，你可以在 Astro 文件中[使用现有的 React 组件](/zh-cn/guides/framework-components/)。但是，如果他们不需要交互，你可能希望将它们转换为 `.astro` 组件！
 
-6. 删除任何 GraphQL 查询。改为使用 import 和 [`Astro.glob()`](/zh-cn/reference/api-reference/#astroglob) 语句来查询你的本地文件。
+6. 删除任何 GraphQL 查询。改为使用 import 和 [`import.meta.glob()`](/zh-cn/reference/api-reference/#importmataglob) 语句来查询你的本地文件。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-gatsby.mdx:128]:
```suggestion
6. 删除任何 GraphQL 查询。改为使用 import 和 [`import.meta.glob()`](/zh-cn/guides/imports/#importmetaglob) 语句来查询你的本地文件。
```

File: src/content/docs/zh-cn/guides/migrate-to-astro/from-nextjs.mdx [hunk 2]
```diff
@@ -115,11 +117,11 @@ Astro 在你项目的根目录有一个配置文件，叫做 [`astro.config.mjs`
 
 3. 将任何必要的 JavaScript，包括导入语句，移到["代码栅栏"（`---`）](/zh-cn/basics/astro-components/#组件脚本)。注意：[条件渲染内容](/zh-cn/reference/astro-syntax/#动态-html)的 JavaScript 通常直接写在 Astro 的 HTML 模板内。
 
-4. 使用 [`Astro.props`](/zh-cn/reference/api-reference/#astroprops) 来访问之前传递给你的 Next 函数的任何额外 props。
+4. 使用 [`Astro.props`](/zh-cn/reference/api-reference/#props) 来访问之前传递给你的 Next 函数的任何额外 props。
 
 5. 决定是否需要将所有导入的组件都转换为 Astro。安装了官方集成后，[你可以在 Astro 文件中使用现有的 React 组件](/zh-cn/guides/framework-components/)。但是，如果它们不需要交互，你可能需要将它们转换为 `.astro` 组件！
 
-6. 用导入语句或 [`Astro.glob()`](/zh-cn/reference/api-reference/#astroglob) 代替 `getStaticProps()` 来查询你的本地文件。使用 `fetch()` 来获取外部数据。
+6. 用导入语句或 [`import.meta.glob()`](/zh-cn/reference/api-reference/#importmetaglob) 代替 `getStaticProps()` 来查询你的本地文件。使用 `fetch()` 来获取外部数据。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/migrate-to-astro/from-nextjs.mdx:124]:
```suggestion
6. 用导入语句或 [`import.meta.glob()`](/zh-cn/guides/imports/#importmetaglob) 代替 `getStaticProps()` 来查询你的本地文件。使用 `fetch()` 来获取外部数据。
```

File: src/content/docs/zh-cn/reference/api-reference.mdx [hunk 3]
```diff
@@ -1,1396 +1,943 @@
...
+  </article>
+))}
+</div>
 ```
 
-另见：[`Astro.redirect()`](#astroredirect)
+`.glob()` 只有一个参数：你想导入的本地文件相对 glob URL。它是异步的，并返回匹配文件的导出数组。
 
-### `context.rewrite()`
+`.glob()` 不接受变量或字符串插值，因为它们无法静态分析。 (请参阅 [导入指南](/zh-cn/guides/imports/#supported-values) 了解解决方法。) 这是因为 `Astro.glob()` 是 Vite 的 [`import.meta.glob()`](https://vite.dev/guide/features.html#glob-import) 的封装。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/api-reference.mdx:865]:
```suggestion
`.glob()` 不接受变量或字符串插值，因为它们无法静态分析。 (请参阅 [导入指南](/zh-cn/guides/imports/#supported-values) 了解解决方法。) 这是因为 `Astro.glob()` 是 Vite 的 [`import.meta.glob()`](https://vite.dev/guide/features.html#支持的值) 的封装。
```

**Review by Nin3lee — APPROVED** [review]
Looks great to me! Absolutely huge one! Thanks again for your effort and contribution!

**Comment by liruifengv** [comment]
Let's ship it! 🚀

**Pull request merged**